.github/workflows/alpine-32bit-build-and-test.yaml

```diff
@@ -74,3 +74,10 @@ jobs:
         name: PostgreSQL log
         path: postmaster.log
 
+    - name: Slack Notification
+      if: failure()
+      env:
+        SLACK_WEBHOOK: ${{ secrets.SLACK_WEBHOOK }}
+        SLACK_TITLE: Regression 32-bit PG${{ matrix.pg }} ${{ job.status }}
```

Comment:
Maybe the title should mention that this is a failure?

Also, shouldn't we have a message that contains, e.g., a link to the build job that failes, git commit, author, etc.? Or is this somehow auto-generated/default? (The current info seems somewhat barebones.)

Reply:
![image](https://user-images.githubusercontent.com/31455525/84256905-dc2f3b80-ab14-11ea-9fa3-0153a6be92f0.png)

Reply:
job.status will be failure